event-stream-service/charts/event-stream-service/README.md

```diff
@@ -0,0 +1,55 @@
+# Event Stream Service Installation
+
+This current documentation will be very simple and make assumptions:
+
+- you are installing on BC Gov Openshift.
+- you have installed `oc` command line tools on your workstation that work on BC Gov Openshift.
+- you know how to get your Openshift token for running your `oc` command line tools.
+- you have installed [helm](https://helm.sh) on your workstation.
+- you have a basic understanding of helm and values files overrides.
+- you have access/permissions in your Openshift namespace to run `helm` and `oc` commands.
+
+**September 3, 2024** - we are using the `a191b5` namespaces to host the proof of concepts. These spaces have minimal resources (we can request more as we learn about the requirements).
+
+## Basic Instructions
+
+1. get your Openshift token
+2. use oc login to your namespace
+3. navigate to the <repo>/event-stream-service
```

Comment:
I think this `<repo>` needs to be in backticks so that it is displayed. Right now it's not shown, possibly because it's being interpreted as HTML.

Reply:
updated